tests/bearlib/aspects/ClassTest.py

```diff
@@ -30,4 +32,23 @@ class SubAspect:
             class docs:
                 example = 'Example'
 
+    def test_issubaspect(self, RootAspect):
+        @RootAspect.subaspect
+        class SubAspect:
```

Comment:
It is also fine to use `SubAspect` fixture here. But again... just a matter of personal taste ;)